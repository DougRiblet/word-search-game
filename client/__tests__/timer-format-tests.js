import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import TimerFormat from '../components/Timer-Clock/timer-format';

describe('Timer Clock', () => {
  test('Snapshot with timer off', () => {
    const TimerFormat_tree = renderer.create(
      <TimerFormat secondsleft={140} />
    ).toJSON();
    expect(TimerFormat_tree).toMatchSnapshot();
  });

  test('renders without crashing with timer off', () => {
    const div = document.createElement('div');
    ReactDOM.render(<TimerFormat secondsleft={140} />, div);
  });

});
