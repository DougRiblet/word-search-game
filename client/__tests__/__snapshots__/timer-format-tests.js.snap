// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer Clock Snapshot with timer off 1`] = `
<span>
  2
  :
  20
</span>
`;
